core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -984,6 +984,40 @@ public ExecResult execInContainer(Charset outputCharset, String... command)
         return result;
     }
 
+    public FrameConsumerResultCallback launchInContainer(Charset outputCharset, String... command)
```

Comment:
there is already `.exec` method, why do we need another one?

Reply:
I'll check it

Reply:
Do you mean `execInContainer`? It's blocking and waiting for the end of the call.
I need one to launch recording and continue testing while recording is running